movie-browser/src/components/Header.js

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from 'react';
+
+class Header extends Component {
```

Comment:
Not a blocker but this can perfectly be a functional component (aka, just a function)